src/org/rust/lang/core/psi/impl/mixin/RustImplMethodImplMixin.kt

```diff
@@ -1,15 +1,13 @@
 package org.rust.lang.core.psi.impl.mixin
 
 import com.intellij.lang.ASTNode
-import com.intellij.psi.PsiElement
+import org.rust.lang.core.psi.RustDeclaringElement
 import org.rust.lang.core.psi.RustImplMethod
-import org.rust.lang.core.psi.RustPatVar
 import org.rust.lang.core.psi.impl.RustNamedElementImpl
-import org.rust.lang.core.psi.util.boundVariables
 
-abstract class RustImplMethodImplMixin(node: ASTNode) : RustNamedElementImpl(node)
-        , RustImplMethod {
+abstract class RustImplMethodImplMixin(node: ASTNode)   : RustNamedElementImpl(node)
+                                                        , RustImplMethod {
```

Comment:
I guess we should settle on some code style...

Whitespace changes are annoing :(